modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js

```diff
@@ -155,36 +177,29 @@ function ResultRankingsAdd({cancelUrl, formName, namespace}) {
 							className="c-ml-1 reference-mark"
 							symbol="asterisk"
 						/>
-
-						<span
-							className={getCN(
-								'input-label-help-icon',
-								PORTAL_TOOLTIP_TRIGGER_CLASS
-							)}
-							data-title={Liferay.Language.get(
-								'result-rankings-scope-help'
-							)}
-						>
-							<ClayIcon
-								className="c-ml-1 text-secondary"
-								symbol="question-circle-full"
-							/>
-						</span>
 					</label>
 
-					<ClayDropDown
-						closeOnClick
+					<ClayButton
+						aria-label={Liferay.Language.get('scope')}
+						className="form-control form-control-select"
+						displayType="unstyled"
+						onClick={_handleScopeDropdownChange}
+						ref={alignElementRef}
+					>
+						{SCOPE_INFO[scopeType].label}
+					</ClayButton>
+
+					<ClayDropDown.Menu
+						active={scopeDropdownActive}
+						alignElementRef={alignElementRef}
 						closeOnClickOutside
-						menuWidth="sm"
-						trigger={
-							<ClayButton
-								aria-label={Liferay.Language.get('scope')}
-								className="form-control form-control-select"
-								displayType="unstyled"
-							>
-								{SCOPE_LABELS[scopeType]}
-							</ClayButton>
-						}
+						onActiveChange={setScopeDropdownActive}
+						style={{
+							maxWidth: '100%',
+							width:
+								alignElementRef.current &&
+								alignElementRef.current.clientWidth + 'px',
```

Comment:
Nice! Thanks for including this. I was wondering about how to get the dropdown width the correct size since the dropdown menu is completely separate in the DOM. 👍

Reply:
Oh thanks! I might have gotten this from one of our existing frontend components with a dropdown 😅